src/drivers/gps/gps.cpp

```diff
@@ -263,6 +263,7 @@ extern "C" __EXPORT int gps_main(int argc, char *argv[]);
 GPS::GPS(const char *path, gps_driver_mode_t mode, GPSHelper::Interface interface, bool fake_gps,
 	 bool enable_sat_info, Instance instance) :
 	_serial_fd(-1),
+	_baudrate(0),
```

Comment:
@bkueng This was your scanning issue. The variable is used as state, but wasn't initialized properly.

Reply:
Well, I had additional local changes. The submodule update needed to be in sync with that.